src/actions/eval-frame-action-validator.js

```diff
@@ -109,7 +123,9 @@ export default function validateActionFromEvalFrame(action) {
     );
   } else if (!Object.keys(schemas).includes(action.type)) {
     throw new ActionSchemaValidationError(
-      "Invalid action from eval frame: action type not permitted"
+      `Invalid action from eval frame: action type not permitted: ${
```

Comment:
I needed to also bring this in because debugging one bit was a bit difficult in this circumstance, but this change should be no surprise